### DISCUSSION TITLE
unintended removal after windows 11 upgrade

### DISCUSSION THREAD

**Discussion by lhpitn** [Open-Shell/Open-Shell-Menu#1986] [Q&A]
Hi,

in our company we are testing the upgrade to windows 11 using a centralized desktop management system. Windows 11 upgrade is working fine so far, the problem is that we cant get rid of open-shell, which is defunct and requests a anoying uac after login and when you say no everything freezes. So we would like some powershell code (or some other stuff that can be execuded unintended through management systems on multiple devices) to reliably remove open shell unintended using remote management. Any idea?

thanks in advance

**Comment by lhpitn** [comment]
see also https://github.com/Open-Shell/Open-Shell-Menu/issues/1987

**Comment by ge0rdi** [comment]
> So we would like some powershell code (or some other stuff that can be execuded unintended through management systems on multiple devices) to reliably remove open shell unintended using remote management. Any idea?

Is there a problem with uninstalling `Open-Shell` in _standard_ way (before upgrading to Win11)? It should be possible to do it via poweshell/cmd.

**Comment by ge0rdi** [comment]
> Windows 11 upgrade is working fine so far, the problem is that we cant get rid of open-shell, which is defunct and requests a anoying uac after login and when you say no everything freezes.

To avoid any user interaction after OS upgrade, please see this guide:
https://github.com/Open-Shell/Open-Shell-Menu/wiki/OS-upgrade-without-user-interaction

**Comment by webpleasure** [comment]
Hi,

I also have the same request. In the company, we have many workstations being upgraded from Win10 to Win11. In some cases, they use Remote Desktop for the connection, and (unfortunately) OpenShell in causing some issues. We would to remove it and then evaluate the installation in the future. 
I would need to know if there is a command line for silent uninstallation.
Thanks.

**EDIT:** I am answering myself thanks to the information obtained with the command " /? " which I too often forget.
The command that worked for me for silent uninstallation is: openshellsetup.exe /x %MSI% /qn

**Comment by nanderer** [comment]
and how to do that?
